var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -379,10 +382,15 @@ def cmake_args(self):
         ]
 
         api = spec.variants['api'].value
+        if sys.platform == 'win32' and self.spec.satisfies('+staticmt'):
```

Comment:
I'm wondering if `hdf5~shared` and `hdf5+staticmt` are redundant: can you make a static library for `hdf5` on Windows that doesn't use this?

Reply:
IIRC, the "regular" build links nonshared with /MD flags, so there is a reasonable variant to be had here

Reply:
I don't think there's a `+staticmt` variant anymore, so I think this `if` needs to be refined.